### DISCUSSION TITLE
Does ESP32-Mini support bluetooth classic (this library)? How to tell which bluetooth modules support A2DP?

### DISCUSSION THREAD

**Discussion by mrhempman69** [pschatzmann/ESP32-A2DP#478] [Q&A] [answered]
Hi,

I was wondering if this module would support this library: https://www.digikey.com/en/products/detail/espressif-systems/ESP32-MINI-1-N4/13532111

It says that it supports "802.11b/g/n, Bluetooth v4.2 +EDR, Class 1, 2 and 3 RF Transceiver Modules and Modems".

In general, how do you determine if an ESP32-xx module supports bluetooth classic by its datasheet?

It's confusing, because the s2 s3 c3 will say "Bluetooth 5.0". Even though it's a higher version number, apparently it doesn't support A2DP.

**Comment by pschatzmann** [comment, marked as answer]
As far as I see, this is a regular ESP32, so it should work.

If you check the technical sheets of the other new ESP32 processor series, they state that they support Bluetooth 5 LE. This is the hint that only Low Energy is supported! e.g. [S3](https://www.espressif.com/sites/default/files/documentation/esp32-s3_datasheet_en.pdf)
